include/dd/Node.hpp

```diff
@@ -15,23 +15,25 @@
 #include <utility>
 
 namespace dd {
+    // NOLINTNEXTLINE(readability-identifier-naming)
     struct vNode {
         std::array<Edge<vNode>, RADIX> e{};    // edges out of this node
         vNode*                         next{}; // used to link nodes in unique table
         RefCount                       ref{};  // reference count
         Qubit                          v{};    // variable index (nonterminal) value (-1 for terminal)
 
-        static vNode            terminalNode;
-        constexpr static vNode* terminal{&terminalNode};
+        static vNode            terminalNode;            // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
+        constexpr static vNode* terminal{&terminalNode}; // NOLINT(cppcoreguidelines-avoid-non-const-global-variables,readability-identifier-naming)
 
         static constexpr bool isTerminal(const vNode* p) { return p == terminal; }
     };
     using vEdge       = Edge<vNode>;
     using vCachedEdge = CachedEdge<vNode>;
 
+    // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
     inline vNode vNode::terminalNode{{{{nullptr, Complex::zero}, {nullptr, Complex::zero}}}, nullptr, 0U, -1};
 
-    struct mNode {
+    struct mNode {                             // NOLINT(readability-identifier-naming)
```

Comment:
Try to stay consistent with `NOLINTNEXTLINE` and `NOLINT`. Linting on the previous struct was ignored differently.